rust/sdk/src/service/events.rs

```diff
@@ -274,7 +274,10 @@ impl<T> std::fmt::Display for EventResponse<T> {
                 write!(f, "composite event")
             }
             EventMeta::Event { key, meta } => {
-                let time = chrono::Local.timestamp_millis(meta.timestamp.as_i64() / 1000);
+                let time = chrono::Local
+                    .timestamp_millis_opt(meta.timestamp.as_i64() / 1000)
+                    .single()
+                    .expect("stored valid");
```

Comment:
these messages are to the reader of the code, not to the person executing the code, since it basically says why the unwrapping cannot possibly fail: we stored the timestamp as serialization of Timestamp, so reading it back ought to work fine

Reply:
@rkuhn